app/src/app/components/contents/runner/run/PacketRunPacketGroupFields.tsx

```diff
@@ -9,10 +9,10 @@ import { PacketRunParamsField } from "./PacketRunParamsField";
 
 interface PacketRunPacketGroupFieldsProps {
   form: UseFormReturn<z.infer<typeof packetRunFormSchema>>;
-  branchCommit: string;
+  branchCommit: string | undefined;
 }
 export const PacketRunPacketGroupFields = ({ form, branchCommit }: PacketRunPacketGroupFieldsProps) => {
-  const { packetGroups, isLoading, error } = useGetRunnerPacketGroups(branchCommit);
+  const { packetGroups, isLoading, error } = useGetRunnerPacketGroups(branchCommit ?? null);
```

Comment:
instead of converting from `undefined` to `null` why not just feed it in as `undefined`?
```suggestion
  const { packetGroups, isLoading, error } = useGetRunnerPacketGroups(branchCommit);
```
and the `use` hooks can accept undefined and still work the same

Reply:
Yeah good point thanks.